src/appleseed/renderer/modeling/frame/frame.h

```diff
@@ -107,9 +108,15 @@ class APPLESEED_DLLSYMBOL Frame
     // Access the AOV images.
     ImageStack& aov_images() const;
 
+    // Access extra AOV indexs.
+    std::vector<size_t>& extra_aov_indexs() const;
```

Comment:
There are several problems with this:

1. One plural of index is indexes (not indexs), the other is indices. Both are fine.
2. What are AOV indexes?
3. Why is it non-const?
4. The deal breaker: We cannot use anything from `std` in DLL-exported classes such as `Frame`.

If we really cannot avoid exposing this, then look into the `DECLARE_ARRAY()` macro to declare a new DLL-safe array.